src/onegov/server/cli.py

```diff
@@ -231,7 +231,7 @@ def run(
             environment=sentry_environment,
             send_default_pii=send_ppi,
             traces_sample_rate=traces_sample_rate,
-            profiles_sample_rate=profiles_sample_rate,
```

Comment:
This seems wrong, pretty sure it should be profiles_sample_rate, did you make sure to update your dependencies locally? Make sure to run `make install` at least once when you run into troubles with pre-commit.

Reply:
I followed to the init of the sentry client and there was no profiles_sample_rate. Let me do the 'make install' and check again...

Reply:
https://docs.sentry.io/platforms/python/profiling/?original_referrer=https%3A%2F%2Fgithub.com%2Fgetsentry%2Fsentry-python#enable-profiling

Reply:
`make install`helped, I also can see the `profiles_sample_rate` in the sentry_sdk.init()